### PR TITLE
Update create_accessions.py to send v2 requests

### PR DESCRIPTION
**Pull request by sgrimm**
Update the test script that creates random accessions so it can optionally do so
using the v2 API endpoints. This allows a limited form of fuzz testing of the v2
API: it doesn't check for correctness but can detect if the request handler bombs
out on what ought to be valid input.

### CODE REVIEW

**Comment by sgrimm** [comment]
Current dependencies on/for this PR:
* main
  * **PR #591** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/591" target="_blank">Graphite</a>  👈
    * **PR #592** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/592" target="_blank">Graphite</a> 
    * **PR #594** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/594" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-server/591?utm_source=stack-comment).

File: scripts/create_accessions.py [hunk 1]
```diff
@@ -220,6 +225,48 @@ def generate_accession(facility_id: int) -> Dict:
...
+    collectors = [generate_staff_responsible() for n in range(num_collectors)] or None
+
+    bag_numbers = generate_bag_numbers()
+    geolocations = (
+        list([generate_geolocation() for x in bag_numbers]) if bag_numbers else None
+    )
+    viability_test_types = [generate_test_type()]
+
+    species = generate_species()
+    family = generate_family(species)
```
Comment (karthikbtf) [inline scripts/create_accessions.py]: unused? we can skip passing species_ids in, if so.

**Review by karthikbtf — APPROVED** [review]
lgtm, one comment

File: scripts/create_accessions.py (hunk 1 above)
Comment (sgrimm) [reply scripts/create_accessions.py]: Quite right. Originally I had the v1 and v2 code in the same functions and I forgot to clean this up when I split them.

Comment (karthikbtf) [reply scripts/create_accessions.py]: Oops. I missed catching that `family` is unused.

**Review by karthikbtf — APPROVED** [review]
Nice!

**Pull request merged**